app/controllers/socure_webhook_controller.rb

```diff
@@ -134,4 +135,27 @@ def user
   def docv_transaction_token
     @docv_transaction_token ||= event[:docvTransactionToken] || event[:docVTransactionToken]
   end
+
+  def repeat_webhook
+    endpoints = IdentityConfig.store.socure_docv_webhook_repeat_endpoints
+    return if endpoints.blank?
+
+    headers = {
+      Authorization: request.headers['Authorization'],
+      'Content-Type': request.headers['Content-Type'],
+    }
+
+    body = socure_params.to_h
+
+    wr = DocAuth::Socure::WebhookRepeater.new(body:, headers:, endpoints:)
+    wr.broadcast
+  rescue => exception
+    NewRelic::Agent.notice_error(
+      exception,
+      custom_params: {
+        event: 'Failed to broadcast Socure webhook',
+        body:,
+      },
+    )
+  end
```

Comment:
it looks like inside the `WebhookRepeater` class we are already rescuing most errors and sending them to NewRelic, does that make this redundant?
```suggestion
  end
```